lib/transloadit/request.rb

```diff
@@ -78,6 +85,19 @@ def post(payload = {})
   end
 
   #
+  # Performs an HTTP PUT to the request's URL. Takes an optional hash
+  # containing the form-encoded payload.
+  #
+  # @param  [Hash] payload the payload to form-encode along with the POST
+  # @return [Transloadit::Response] the response
+  #
+  def put(payload = {})
+    self.request! do
+      self.api[url.path].put(self.to_payload(payload), API_HEADERS)
```

Comment:
Talking about `API_HEADERS`, I prefer to use the double quote syntax instead of `%{}` for simple string interpolation. People tend to be confused by this syntax and it doesn't give us anything there :)